static/js/ui_init.js

```diff
@@ -178,7 +178,7 @@ function initialize_right_sidebar() {
     $("#user_presences").on("mouseenter", ".user_sidebar_entry", (e) => {
         const $status_emoji = $(e.target).closest(".user_sidebar_entry").find("img.status_emoji");
         if ($status_emoji.length) {
-            const animated_url = $status_emoji.attr("data-animated-url");
+            const animated_url = CSS.escape($status_emoji.attr("data-animated-url"));
```

Comment:
I tested this, and I'm pretty sure this ends up double-escaping the content and making this not work. It's already been escaped when storing it in` data-animated-url` in the first place, so I think the GitHub warning is incorrect.